app/utils/validators/forms/FieldValidator.scala

```diff
@@ -51,11 +50,11 @@ object FieldValidator {
 
   val isPresent: Option[_] => Boolean = _.nonEmpty
 
-  val isMissing: Option[_] => Boolean = _.isEmpty
+  val isNone: Option[_] => Boolean = _.isEmpty
```

Comment:
Nit: If changing this name then `isPresent` should be renamed to `isSome` for consistency :+1: